src/stories/Editor.stories.svelte

```diff
@@ -1,16 +1,20 @@
 <script>
-  //
   import Editor from '../main/component/Editor.svelte';
-  import { Meta, Story } from '@storybook/addon-svelte-csf';
+  import { Story } from '@storybook/addon-svelte-csf';
+
+  export const meta = {
```

Comment:
I'm getting the `Cannot read properties of undefined (reading 'Iframe')` error when running `yarn storybook`

Looking at the examples in the README at https://github.com/storybookjs/addon-svelte-csf, the meta and Editor look to be required in the `<script context="module"` tag instead.

```
<script context="module">
  import Button from './Button.svelte';

  export const meta = {
    title: "Button",
    component: Button
  }
</script>
```

Reply:
Ah whoops, my bad. Fixed that up